components/member-profile/active-task/index.js

```diff
@@ -1,19 +1,62 @@
 import PropTypes from 'prop-types';
-import classNames from 'components/member-profile/active-task/active-task.module.scss';
-import { timeWas } from 'helper-functions/time-was';
+import classNames from './active-task.module.scss';
+import { timeWas } from '../../../helper-functions/time-was';
+import { percentageofDaysRemaining } from '../../../helper-functions/taskProgress';
+
 const ActiveTask = ({ taskDetails }) => {
-  const { title, purpose, startedOn, endsOn } = taskDetails;
-  let completedDate = '';
+  const { title, purpose, startedOn, endsOn, percentCompleted } = taskDetails;
+  let completedDate,
+    percentageOfDaysRemaining = '';
   completedDate = timeWas(startedOn * 1000, false, endsOn * 1000);
+
+  let percentOfTaskLeft = 100 - percentCompleted;
+
+  percentageOfDaysRemaining = percentageofDaysRemaining(startedOn, endsOn, completedDate);
```

Comment:
It is better to go with const here rather than initializing with empty string and then assigning a value